docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
     stdin_open: true
     env_file:
       - ssh-keys.env
-    command: vendor/bin/phpunit
+    command: composer ci
```

Comment:
I think you can leave in this library only one service with mounted volume. Please check this PR with similar improvements https://github.com/keboola/db-writer-common/pull/13/files#diff-4e5e90c6228fd48698d074241c2ba760

Also readme is not up to date. Currently it throws:
```
 docker-compose run --rm dev 
Starting dbextractorcommon_mysql_1 ... done
Starting dbextractorcommon_sshproxy_1 ... done
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Do not run Composer as root/super user! See https://getcomposer.org/root for details
./composer.json is valid
> phpcs -n --ignore=vendor --extensions=php .
sh: 1: phpcs: not found
Script phpcs -n --ignore=vendor --extensions=php . handling the phpcs event returned with error code 127
Script @phpcs was called via build
Script @build was called via ci
```